packages/app-metadata-controller/src/AppMetadataController.test.ts

```diff
@@ -0,0 +1,154 @@
+import { Messenger } from '@metamask/base-controller';
+import AppMetadataController, {
+  getDefaultAppMetadataControllerState,
+  type AppMetadataControllerOptions,
+} from './AppMetadataController';
```

Comment:
Since we took away the default export, should we change this to:

``` suggestion
import {
  AppMetadataController,
  getDefaultAppMetadataControllerState,
  type AppMetadataControllerOptions,
} from './AppMetadataController';
```